src/_gettsim/functions/policy_function.py

```diff
@@ -37,59 +39,29 @@ class PolicyFunction(Callable):
 
     def __init__(  # noqa: PLR0913
         self,
-        function: Callable,
         *,
-        module_name: str = "",
-        function_name: str | None = None,
-        start_date: date | None = None,
-        end_date: date | None = None,
+        function: Callable,
+        leaf_name: str,
+        start_date: date = DEFAULT_START_DATE,
```

Comment:
Sorry, my bad. With "the above syntax" I meant calling the decorator like `dec(dec_args)(func)`, not defining defaults. We'd have to do the same for the other three... And you want to see the defaults right in the interface, e.g., when hovering over a use of it as decorator.